forge-gui/res/cardsfolder/upcoming/zur_eternal_schemer.txt

```diff
@@ -0,0 +1,10 @@
+Name:Zur, Eternal Schemer
+ManaCost:W U B
+Types:Legendary Creature Human Wizard
+PT:1/4
+K:Flying
+S:Mode$ Continuous | Affected$ Creature.Enchantment+YouCtrl | AddKeyword$ Deathtouch & Lifelink & Hexproof | Description$ Enchantment creatures you control have deathtouch, lifelink, and hexproof.
+A:AB$ Animate | Cost$ 1 W | ValidTgts$ Enchantment.nonAura+YouCtrl | TgtPrompt$ Select target non-Aura enchantment you control | Power$ X | Toughness$ X | Types$ Creature | Duration$ Permanent | SpellDescription$ Target non-Aura enchantment you control becomes a creature in addition to its other types and has base power and toughness each equal to its mana value.
```

Comment:
how is the stack description for this ability? Something tells me it will need a `StackDescription$`

Reply:
Added a `StackDescription$` that will reflect the `SpellDescription$` verbatim. Also fixed a missing activated ability cost in the Oracle text